app/overlays/lnl/module_overlay.conf

```diff
@@ -1,2 +1,3 @@
+CONFIG_LIBRARY_BASE_ADDRESS=0xa0688000
```

Comment:
Why here as well, shouldn't the definition in board conf file be enough?

Reply:
@kv2019i yes, thanks, I noticed that already too, will fix in the next version